### PR TITLE
7903414: Support test.query for TestNG tests.

### PR DESCRIPTION
**Pull request by lahodaj**
Tried to add the `test.query` support for TestNG tests.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [CODETOOLS-7903414](https://bugs.openjdk.org/browse/CODETOOLS-7903414): Support test.query for TestNG tests.

### Reviewers
 * [Jonathan Gibbons](https://openjdk.org/census#jjg) (@jonathan-gibbons - **Reviewer**) ⚠️ Review applies to [60c86e29](https://git.openjdk.org/jtreg/pull/145/files/60c86e29b9493cd64d0cdf572e5f825da31ad94e)
 * [Christian Stein](https://openjdk.org/census#cstein) (@sormuras - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jtreg pull/145/head:pull/145` \
`$ git checkout pull/145`

Update a local copy of the PR: \
`$ git checkout pull/145` \
`$ git pull https://git.openjdk.org/jtreg pull/145/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 145`

View PR using the GUI difftool: \
`$ git pr show -t 145`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jtreg/pull/145.diff">https://git.openjdk.org/jtreg/pull/145.diff</a>

</details>

### CODE REVIEW

File: test/testngQueryTest/a/b/c/TEST.properties [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+# or visit www.oracle.com if you need additional information or have any
+# questions.
+#
+
+TestNG.dirs = .
```
Comment (jonathan-gibbons) [inline test/testngQueryTest/a/b/c/TEST.properties:24]: Add newline?

**Review by jonathan-gibbons — APPROVED** [review]
OK, but please add final newlines to files where one is missing.

**Review by sormuras — APPROVED** [review]

**Pull request closed without merging**